site/_scss/blocks/_toc.scss

```diff
@@ -30,6 +30,35 @@
   transform: rotate(270deg);
 }
 
+toc-active {
+  display: block;
+  max-height: calc(100vh - #{get-size(200)} * 2);
+  overflow-x: hidden;
+  overflow-y: auto;
+  position: sticky;
+  top: get-size(200);
+
+  &::-webkit-scrollbar {
+    height: 8px;
+    width: 8px;
+  }
+  &::scrollbar {
+    height: 8px;
+    width: 8px;
+  }
+
+  &::-webkit-scrollbar-thumb {
+    background: rgba(128,134,139,.26);
```

Comment:
on web.dev we use $GREY_500 for scrollbars. On d.c.c. we tend to use custom properties for colors so we can (eventually) support a dark mode theme. It would be good to add this color to our theme and use the custom property in case we want to change our custom scrollbar colors. https://github.com/GoogleChrome/developer.chrome.com/blob/main/site/_scss/_theme.scss